src/main/java/com/projectkorra/core/command/AbilitiesCommand.java

```diff
@@ -0,0 +1,77 @@
+package com.projectkorra.core.command;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import org.bukkit.command.CommandSender;
+
+import com.projectkorra.core.ability.Ability;
+import com.projectkorra.core.ability.AbilityManager;
+import com.projectkorra.core.ability.type.Bindable;
+import com.projectkorra.core.skill.Skill;
+
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.ClickEvent;
+import net.md_5.bungee.api.chat.ComponentBuilder;
+import net.md_5.bungee.api.chat.ComponentBuilder.FormatRetention;
+import net.md_5.bungee.api.chat.HoverEvent;
+import net.md_5.bungee.api.chat.HoverEvent.Action;
+import net.md_5.bungee.api.chat.hover.content.Text;
+
+public class AbilitiesCommand extends PKSubCommand {
+
+	public AbilitiesCommand() {
+		super("abilities", "Show the abilities for a skill", "/pk abilities <skill>", Arrays.asList("abils", "display", "list", "catalog"));
+	}
+
+	@Override
+	public void execute(CommandSender sender, String[] args) {
```

Comment:
Similar to abilities, can we add a language/chat decorator class so we can lessen the bulk of the commands, add more customization, and adhere to single responsibility.